src/librustc_borrowck/diagnostics.rs

```diff
@@ -286,6 +286,54 @@ You can read more about cell types in the API documentation:
 https://doc.rust-lang.org/std/cell/
 "##,
 
+E0389: r##"
+An attempt was made to assign to data in a non-mutable reference. Example:
```

Comment:
Okay. In that case, do you want to modify the RFC to change line 86 to say "Example of erroneous code:"?